addons/website_sale/views/snippets.xml

```diff
@@ -34,4 +34,66 @@
     </xpath>
 
 </template>
+
+<template id="remove_external_snippets" inherit_id="website.external_snippets">
+    <xpath expr="//t[@t-install='website_sale']" position="replace"/>
+</template>
+
+<template id="website_sale_snippet" inherit_id="website.snippets">
+    <xpath expr="//div[@id='snippet_feature']//t[@t-snippet][last()]" position="after">
+        <t t-snippet="website_sale.s_products_searchbar" t-thumbnail="/website/static/src/img/snippets_thumbs/s_products_searchbar.png"/>
+    </xpath>
+    <xpath expr="//div[@id='snippet_content']//t[@t-snippet][last()]" position="after">
+        <t t-snippet="website_sale.o_wsale_products_searchbar_input" t-thumbnail="/website/static/src/img/snippets_thumbs/s_products_searchbar.png"/>
+    </xpath>
+</template>
+
+<template id="website_sale_snippet_options" inherit_id="website.snippet_options">
+    <xpath expr="." position="inside">
+        <div data-js="ProductsSearchBar"
+            data-selector="div.o_wsale_products_search_bar_input">
+            <a tabindex="-1" href="#" class="dropdown-item" data-no-preview="true" data-open-searchbar-settings=""><i class="fa fa-search"></i>Products Search Bar</a>
+        </div>
+    </xpath>
+    <xpath expr="//div[@data-js='content']" position="attributes">
+        <attribute name="data-selector" add=".o_wsale_products_searchbar_input" separator=","/>
+        <attribute name="data-drop-near" add=".o_wsale_products_searchbar_input" separator=","/>
+    </xpath>
+</template>
+
+<!-- Products Search Bar input-group template -->
+<template id="website_sale_search_input_group" name="Products Search Bar Input">
+    <div class="input-group o_wsale_products_search_bar_input">
+        <form>
```

Comment:
I really really doubt a a form between and input group and its elements will produce non broken design.

Anyways, the form has no attribute so it is useless, add the attribute of the current one in the shop....